CameraFollow.cs

```diff
@@ -0,0 +1,25 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class CameraFollow : MonoBehaviour
+{
+    [SerializeField] Transform target;
+    Vector3 offset;
+    Vector3 tempPos;
+    void Start() => offset = transform.position - target.position;
```

Comment:
Jeżeli `target` może być `null` w  `LateUpdate`, to również tutaj może być `null`, nie?